pkg/datapath/linux/requirements.go

```diff
@@ -219,7 +219,7 @@ func CheckMinRequirements() {
 	if !option.Config.DryMode {
 		probeManager := probes.NewProbeManager()
 		if err := probeManager.SystemConfigProbes(); err != nil {
-			log.WithError(err).Warning("BPF system config check: NOT OK.")
+			log.WithError(err).Info("BPF system config check: NOT OK.")
```

Comment:
Shouldn't we inspect the error, and then decide whether the kernel config was unavailable?

Reply:
I agree. I think we should create a specific error for lack of kernel config (`var ErrKernelConfig = errors.New("...")`?) and then check for it here.

I would be even tempted to change log level to Fatal if the error is **not** about lack of kernel config, but rather about some speficic parameter missing. ;)

Reply:
I think this makes sense, I have updated the pull request. Can you take a look and let me know if this is what we are looking for?